test/WebSites/SimpleWebSite/Controllers/HomeController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
I'm going to try to find an extension which will auto-add headers so I stop making this mistake.